01week/javascripting/data-types.js

```diff
@@ -0,0 +1,61 @@
+
+function returnDate(){
+  const date=new Date();
+  return date;}
+
+console.log(returnDate())
+
+
+function NtoS() {
+  const num = 7;
+  const n = num.toString();
+  return (n);
+}
+
+console.log(NtoS());
+
+function stuff(){
+  const s = parseInt('77');
+  return (s);
+}
+console.log(stuff());
+
+
+function add(){
+  const sum = 3+4;
+  return (sum);
+}
+console.log (add());
+
+
+
+function twotrue(){
+  const x=5
+  const y=5
+  if (x===y){
```

Comment:
This function evaluates if x is equal to y, I need it to evaluate if x evaluates to true and y evaluates to true.